go-controller/pkg/node/default_node_network_controller.go

```diff
@@ -614,6 +614,37 @@ func getOVNSBZone() (string, error) {
 	return dbZone, nil
 }
 
+/** HACK BEGIN **/
+// TODO(tssurya): Remove this HACK a few months from now.
+// checkOVNSBNodeLRSR returns true if the logical router static route for the
+// the given nodeSubnet is present in the SBDB
+func checkOVNSBNodeLRSR(nodeSubnet string) bool {
+	match := fmt.Sprintf("match=\"reg7 == 0 && ip4.dst == %s\"", nodeSubnet)
+	stdout, stderr, err := util.RunOVNSbctl("--bare", "--columns", "_uuid", "find", "logical_flow", match)
+	klog.Infof("Upgrade Hack: checkOVNSBNodeLRSR for node - %s : match %s : stdout - %s : stderr - %s : err %v", nodeSubnet, match, stdout, stderr, err)
+	return err == nil && stderr == "" && stdout != ""
+}
+
+// lbExists returns true if the OVN load balancer for the corresponding namespace/name
+// was created
+func lbExists(namespace, name string) bool {
+	stdout, stderr, err := util.RunOVNSbctl("--bare", "--columns", "name", "find", "Load_Balancer")
+	stitchedServiceName := "Service_" + namespace + "/" + name
+	match := strings.Contains(stdout, stitchedServiceName)
+	klog.Infof("Upgrade Hack: lbExists for service - %s/%s/%s : match - %v : stdout - %s : stderr - %s : err %v",
```

Comment:
printing stdout here is printing all the load balancers and flooding the log

Reply:
yeah was meant for debugging but removed it now for per-service.. - now its not done per poll loop which should be less flooding.